bindings/wasm/src/wasm_document.rs

```diff
@@ -236,11 +236,7 @@ impl WasmDocument {
 
   #[wasm_bindgen(js_name = resolveKey)]
   pub fn resolve_key(&mut self, query: &str) -> Result<WasmVerificationMethod, JsValue> {
```

Comment:
@l1h3r previously this code relied on `IotaVerificationMethod::try_from_core()` (because `IotaDocument.resolve()` returns a `VerificationMethod`), should we also change the signature of this to just return `WasmVerificationMethod`? It should be impossible for this to fail now (since `WasmDocument` wraps an already verified `IotaDocument`. lmk what you think.

Can also change the signature in a follow-up PR since that may be more involved.

Reply:
The conversion doesn't fail but the query can still fail...but it probably makes more sense to just use `resolve` and return `null` in JavaScript.

This is probably best to do in a separate PR; may try to remove some other `Result`s that don't offer much value in JavaScript.

Reply:
SG, will address in a follow-up.